tutorials/io/plot_20_reading_eeg_data.py

```diff
@@ -128,31 +130,33 @@
 EGI simple binary (.egi)
 ========================
 
-EGI simple binary files can be read in using :func:`mne.io.read_raw_egi`.
-The EGI raw files are simple binary files with a header and can be exported
-from using the EGI Netstation acquisition software.
+EGI simple binary files can be read using :func:`mne.io.read_raw_egi`.
+EGI raw files are simple binary files with a header and can be exported by the
+EGI Netstation acquisition software.
 
 
 .. _import-mff:
 
 EGI MFF (.mff)
 ==============
-These files can also be read with :func:`mne.io.read_raw_egi`.
+
+EGI MFF files can be read with :func:`mne.io.read_raw_egi`.
 
 
 .. _import-set:
 
 EEGLAB set files (.set)
 =======================
 
-EEGLAB .set files can be read in using :func:`mne.io.read_raw_eeglab`
-and :func:`mne.read_epochs_eeglab`.
+EEGLAB .set files can be read using :func:`mne.io.read_raw_eeglab` and
+:func:`mne.read_epochs_eeglab`.
```

Comment:
would it be worthwhile to also add `.fdt` here? I know that it's "more or less" no longer being advertised, but EEGLAB still supports exporting set+fdt ... and MNE-Python can read it.

So if a user uses the "search" function of the docs with `fdt`, they should end up here.

Reply:
Yes, good point!